android/app/src/main/java/org/pathcheck/covidsafepaths/exposurenotifications/reactmodules/ExposureNotificationsModule.java

```diff
@@ -70,7 +70,7 @@ public void onFailure(@NotNull Throwable exception) {
   public void getCurrentENPermissionsStatus(final Promise promise) {
     ExposureNotificationClientWrapper.get(getReactApplicationContext())
         .isEnabled()
-        .addOnSuccessListener(enabled -> promise.resolve(Util.getEnStatusWritableArray(enabled)))
+        .addOnSuccessListener(enabled -> promise.resolve("Active"))
```

Comment:
This should be changed to `promise.resolve(enabled ? "Active" : "Disabled")`

Reply:
Thanks!